server/models/Category.model.js

```diff
@@ -0,0 +1,14 @@
+const mongoose = require('mongoose');
+
+const categorySchema = new mongoose.Schema({
+  category: {
+    type: String,
+    trim: true,
+    uppercase: true,
```

Comment:
Any reason in particular why categories should be in all caps?

Reply:
Standardization purposes of category strings

Reply:
If categories are specific to each user there isn't much use in standardizing. If anything, it reduces the customizability of categories as things like acronyms would be hard to differentiate from regular words. We should try not to take away from personalization options in case users actually want to use lower case letters (I know I would). What users input is what they should get: which is how it should be. Any time this isn't the case without good reason is a little jarring.

Reply:
I mainly wanted to avoid the case where users can add "water" and "Water" as categories. That can introduce unintentional multiple categories, where the user might have expected a single "water" category.

Reply:
That would be on the user, and given that categories are searched up as the user types something like that would hopefully be prevented

Reply:
Ok. I will make this adjustment